setup.py

```diff
@@ -16,4 +16,9 @@ def run(self):
                 self.copy_file('labscript-suite.pth', path)
 
 
-setup(cmdclass={'develop': develop_command})
+VERSION_SCHEME = {
+    "version_scheme": os.getenv("SCM_VERSION_SCHEME", "release-branch-semver"),
```

Comment:
This can probably just be `"version_scheme": "release-branch-semver"`, as:
* `SCM_LOCAL_SCHEME` is used for `local_scheme` only so the local version is removed (required for uploading to test PyPI), i.e. using `no-local-version` rather than `node-and-date`; and
* `SCM_VERSION_SCHEME` _was_ used as `release_branch_semver` required using a git-installed `setuptools_scm` in the GitHub action prior to version `4.1.0`.